applications/openshift/registry/ocp_insecure_registries/rule.yml

```diff
@@ -0,0 +1,53 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: "Check if any insecure registry sources is configured"
+
+description: |-
+    The configuration <tt>registrySources.insecureRegistries</tt> determines the
+    insecure registries that the OpenShift container runtime can access for builds
+    and pods. This configuration setting is for accessing the configured registries
+    without TLS validation which could lead to security breaches and should be
+    avoided.
+    Remove any insecureRegistries configured using the following command:
+    <pre>oc patch image.config.openshift.io cluster --type=json -p "[{'op': 'remove', 'path': '/spec/registrySources/insecureRegistries'}]"</pre>
+
+    For more information, follow
+    {{{ weblink(link="https://docs.openshift.com/container-platform/latest/openshift_images/image-configuration.html",
+                text="the relevant documentation") }}}.
+
+rationale: |-
+    Insecure registries should not be configured, which would restrict the possibilities of
+    OpenShift container runtime accessing registries which cannot be validated.
+
+severity: medium
+
+identifiers:
+  cce@ocp4: CCE-86023-9
+
+references:
+    nist: CM-7(2),CM-7(4),CM-7(5),CM-11
```

Comment:
CM-* here deal with software installation and execution. It does not relate to secure/insecure communication. A relook might be required for this.

Reply:
Nice catch. I would link `CM-5(3)` instead.

Reply:
comment incorporated.